web/profiles/custom/yalesites_profile/modules/custom/ys_toolbar/src/ToolbarItemsService.php

```diff
@@ -124,6 +137,11 @@ public function addItems(): array {
       );
     }
 
+    $this->toolbarItems['toolbar_theme_settings_link'] = $this->buildOffCanvasButton(
```

Comment:
We got some cross module calls. Let's either:
1. List ys_themes as a dependency of the ys_toolbar module
2. Or create a separate service in ys_themes so it contains its own code

Reply:
Added `ys_themes` as a dependency, good call.